src/NuGet.Core/NuGet.Protocol/Providers/ServiceIndexResourceV3Provider.cs

```diff
@@ -141,6 +141,12 @@ public ServiceIndexResourceV3Provider()
                             log,
                             token);
                     }
+                    catch (OperationCanceledException ex)
+                    {
+                        var message = ExceptionUtilities.DisplayMessage(ex);
+                        log.LogInformation(message);
```

Comment:
Won't this just log "A task was cancelled"?  How is that useful to the customer?

Reply:
We were already logging it. I don't know why this method has so much logging, but didn't have good reason to reduce logging with this PR.

Reply:
What do you think about
```suggestion
                         ProtocolDiagnostics.RaiseEvent(new ProtocolDiagnosticHttpEvent(...
```

Reply:
If the request is already having an event raised somewhere else, it will cause us to double count the request, and we'll have wrong data. If the request is not currently having an event raised anywhere else, then we're missing data and this would be a great change to correct that.  Unfortunately these details of exactly how and where all the events are raised are too complex for me to remember, so it would require an investigation to figure out.

Reply:
I would rather keep the behavior as-is, since this PR was really addressing propagation of the error. The investigation could be noted in a new Issue.
If you want to argue we should stop logging the cancellation, I could be persuaded to do that.

Reply:
I did change from LogMinimal to LogInformation... it seemed to me that if I'm logging execution of a command, and there's a cancellation, I expect my log to tell me that. But throwing probably results in the same behavior, eventually.

Reply:
LogMinimal is a higher verbosity than LogInformation, so it logs more often than Information. 

Was that your intention?

Reply:
Nope, I can keep it as LogMinimal, then.

Reply:
Switched to LogMinimal and added a comment-block. This resolved?